__tests__/route-mappers.ts

```diff
@@ -203,17 +201,16 @@ describe('Route mapper', () => {
         const response = await server.inject({
             method: 'POST',
             url: '/mappers/' + sourceId,
-            query: context.params,
+            query: context.params as {[key: string]: string;},
```

Comment:
why do you need this as?

Reply:
if the ContextRequest params options match the same type maybe you can avoid the as.

Reply:
because this line was write before its type... xDDD